developer-workflow/psrt.rst

```diff
@@ -0,0 +1,154 @@
+  No details about the vulnerability report need to be shared with the PSF CNA for a CVE ID to be reserved.
+
+* If the report is a vulnerability, the coordinator will determine the severity of the vulnerability. Severity is one of:
+  **Low**, **Medium**, **High**, and **Critical**. Coordinators can use their knowledge of the code, how the code is likely used,
+  or another mechanism like CVSS for determining a severity. Add this information to the GitHub Security Advisory.
+
+* Once a CVE ID is assigned, the coordinator will share the acceptance and CVE ID with the reporter.
+  Use this CVE ID for referencing the vulnerability. The coordinator will ask the reporter
+  if the reporter would like to be credited publicly for the report and if so, how they would like to be credited.
+  Add this information to the GitHub Security Advisory.
+
+* The coordinator authors the vulnerability advisory text. The advisory must include the following information:
+
+  * Title should be a brief description of the vulnerability and affected component
+    (for example, "Buffer over-read in SSLContext.set_npn_protocols()")
+
+  * Short description of the vulnerability, impact, and the conditions where the affected component is vulnerable, if applicable.
+
+  * Affected versions. This could be "all versions", but if the vulnerability exists in a new feature
+    or removed feature then this could be different. Include versions that are end-of-life in this calculation.
+    (for example, "Python 3.9 and earlier", "Python 3.10 and later", "all versions of Python")
+
+  * Affected components and APIs. The module, function, class, or method must be specified so users can
+    search their codebase for usage. For issues affecting the entire project, this can be omitted.
+
+  * Mitigations for the vulnerability beyond upgrading to a patched version, if applicable.
+
+  This can all be done within the GitHub Security Advisory UI for easier collaboration between reporter and coordinator.
+
+* The coordinator determines the fix approach and who will provide a patch. Some reporters are willing to provide or collaborate to create a
+  patch, otherwise relevant core developers can be invited to collaborate by the coordinator.
+
+   * For **Low** and **Medium** severity vulnerabilities it is acceptable to develop a patch in public.
+     The pull request must be marked with the ``security`` and ``release-blocker`` labels so that a release
+     is not created without including the patch.
+
+   * For **High** and **Critical** severity vulnerabilities the patch must be developed privately using GitHub Security Advisories'
+     "Private Forks" feature. Core developers can be added to the GitHub Security Advisory via "collaborators" to work
+     on the fix together. Once a patch is approved privately and tested, a public issue and pull request can be created
+     with the ``security`` and ``release-blocker`` labels.
```

Comment:
The two sub-items are not indented properly and are rendered improperly.  I fixed the indentation:
```suggestion
* The coordinator determines the fix approach and who will provide a patch.
  Some reporters are willing to provide or collaborate to create a patch,
  otherwise relevant core developers can be invited to collaborate by
  the coordinator.

  * For **Low** and **Medium** severity vulnerabilities it is acceptable
    to develop a patch in public.
    The pull request must be marked with the ``security`` and ``release-blocker``
    labels so that a release is not created without including the patch.

  * For **High** and **Critical** severity vulnerabilities the patch must be
    developed privately using GitHub Security Advisories' "Private Forks" feature.
    Core developers can be added to the GitHub Security Advisory via "collaborators"
    to work on the fix together. Once a patch is approved privately and tested,
    a public issue and pull request can be created with
    the ``security`` and ``release-blocker`` labels.
```

I also rewrapped the text (without changing the content), so that it's more readable (both in an editor and in the diffs.  The rewrapping could be applied to the rest of the document too.